Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
Interesting. I guess we get away with it because it's an "internal" key and no one who gets the contents of the Info.plist is supposed to see it.

Frankly, it's wrong. We should stash that URL inside the CFBundle struct instead. What that means is that _CFBundleCopyInfoDictionaryInDirectoryWithVersion should return the final URL (probably a byref arg), and CFBundleGetInfoDictionary should put it inside the CFBundle struct.